src/pages/distroSettings/tabs/BaseTab.tsx

```diff
@@ -15,12 +21,22 @@ export const BaseTab = <T extends WritableDistroSettingsType>({
   initialFormState,
   ...rest
 }: BaseTabProps<T>) => {
```

Comment:
If you'd like, I think it would be possible to overwrite the user's permissions in Cypress like what is done [here](https://github.com/evergreen-ci/spruce/blob/9b09e0609384d0a039165966bf933c92df5c8a60/cypress/integration/nav_bar.ts#L123), so that you can test the view-only UI in Cypress

P.S. I created [EVG-21024](https://jira.mongodb.org/browse/EVG-21024)